tendrl-node-agent.spec

```diff
@@ -75,8 +85,12 @@ py.test -v tendrl/node-agent/tests || :
 %dir %{_sysconfdir}/tendrl/node-agent
 %dir %{_datadir}/tendrl/node-agent
 %dir %{_sharedstatedir}/tendrl
+%attr(0655, root, root) %{_sysconfdir}/collectd_template/*
+%attr(0655, root, root) %{_libdir}/collectd/*
+
 %doc README.rst
 %license LICENSE
+%{_bindir}/config_manager
```

Comment:
@TimothyAsir you have copied this file as "tendrl_monitoring_config_manager" and now declaring it as "config_manager" , which is wrong